realm/realm-library/src/main/java/io/realm/RealmSchema.java

```diff
@@ -246,12 +248,23 @@ final RealmObjectSchema removeFromClassNameToSchemaMap(String name) {
         return dynamicClassToSchema.remove(name);
     }
 
+    final OsKeyPathMapping getKeyPathMapping() {
```

Comment:
Should this trigger creation first time it is called? Or it is intentionally left as a _manual_ step?

Reply:
It is a manual step on purpose since I can then make sure it is called as part of opening the Realm. While I haven't measured the performance, it is slightly heavyweight as it as to loop through all classes and properties, so it seemed safer to have it as part of the startup cost of opening a Realm.